src/System Application/App/AI/src/Copilot/CopilotCapabilityImpl.Codeunit.al

```diff
@@ -442,6 +443,28 @@ codeunit 7774 "Copilot Capability Impl"
             GuidedExperience.ResetAssistedSetup(ObjectType::Page, Page::"Copilot AI Capabilities");
     end;
 
+    procedure DeactivateCapability(var CopilotSettingsLocal: Record "Copilot Settings")
+    var
+        CopilotDeactivate: Page "Copilot Deactivate Capability";
+        ALCopilotFunctions: DotNet ALCopilotFunctions;
+    begin
+        if ALCopilotFunctions.IsCopilotFeedbackEnabled() then begin
+            CopilotDeactivate.SetCaption(Format(CopilotSettingsLocal.Capability));
+            if CopilotDeactivate.RunModal() = Action::OK then begin
+                CopilotSettingsLocal.Status := CopilotSettingsLocal.Status::Inactive;
+                CopilotSettingsLocal.Modify(true);
+
+                SendDeactivateTelemetry(CopilotSettingsLocal.Capability, CopilotSettingsLocal."App Id", CopilotDeactivate.GetReason());
+                Session.LogAuditMessage(StrSubstNo(CopilotFeatureDeactivatedLbl, CopilotSettingsLocal.Capability, CopilotSettingsLocal."App Id", UserSecurityId()), SecurityOperationResult::Success, AuditCategory::ApplicationManagement, 4, 0);
+            end;
+        end else begin
+            CopilotSettingsLocal.Status := CopilotSettingsLocal.Status::Inactive;
+            CopilotSettingsLocal.Modify(true);
+
+            Session.LogAuditMessage(StrSubstNo(CopilotFeatureDeactivatedLbl, CopilotSettingsLocal.Capability, CopilotSettingsLocal."App Id", UserSecurityId()), SecurityOperationResult::Success, AuditCategory::ApplicationManagement, 4, 0);
```

Comment:
Line 458 and this are the same. Move these to the end, so between 456/466 outside of the if/else scope

Reply:
This is by design. The page could be closed without actually decativation